networks/ghostnet/values.yaml

```diff
@@ -103,7 +103,7 @@ images:
 # We give:
 accounts:
   oxheadbaker:
-    # tz1foXHgRzdYdaLgX6XhpZGxbBv42LZ6ubvE
```

Comment:
Shouldn't this and for some of the other networks not change @nicolasochem ?

Reply:
I changed it back on main.... my goal was to ensure I wouldn't be double baking as teztnets was running in shadow mode on DO. Now that it's migrated, we are back to this address.